code/qcommon/q_shared.h

```diff
@@ -80,6 +80,10 @@ along with this program; if not, see <http://www.gnu.org/licenses/>.
 #define HOMEPATH_NAME_MACOSX HOMEPATH_NAME_WIN
 #endif
 
+#ifndef SOURCE_DATE
```

Comment:
Can you add this to the `shared/qcommon/` folder instead? Perhaps in `q_platform.h`  if you think it fits there.

I want to try to avoid duplication of code between the `code/` and `codemp/` folders as much as possible.